x/feegrant/handler.go

```diff
@@ -23,8 +23,9 @@ func NewHandler(k Keeper) sdk.Handler {
 }
 
 func handleGrantFee(ctx sdk.Context, k Keeper, msg MsgGrantFeeAllowance) (*sdk.Result, error) {
-	grant := FeeAllowanceGrant(msg)
-	k.GrantFeeAllowance(ctx, grant)
+	feegrant := FeeAllowanceGrant{Granter: msg.Granter, Grantee: msg.Grantee, Allowance: msg.Allowance}
```

Comment:
It would be nice to still allow a message (MsgGrantFeeAllowance) to be passed from which we can get a `FeeAllowanceGrant`.

Reply:
We can do that actually, this change is unintended. Will update